app/store/aliases.ts

```diff
@@ -0,0 +1,28 @@
+import * as axios from "axios";
+import {fetchSuccess, fetchFailure, handleRespTextChange} from "./../actions";
+
+const fetchDataAlias = (payload: object) => {
+  return dispatch => {
+    const {method, url, requestId, requestHeaders} = payload.requestDetails;
+    const requestHeadersObject = requestHeaders.reduce((accumulatedObj, reqHeaderNameValuePair) => {
```

Comment:
can you explain what's the intent of this?

Reply:
In axios, if we need to pass the request headers while making a request, we need to specify it this way `headers: {'X-Requested-With': 'XMLHttpRequest'},`.  But from the background script, we get the headers in the following format.
```js
requestHeaders : {
[0] : {name : 'X-Requested-With' , value : 'XMLHttpRequest'},
[1] : ...
}
So the function returns the headers in the way that axios requires.

Ref: https://github.com/axios/axios#request-config